### PR TITLE
3098 Refining documentations for moving_camera_scene module

### PR DESCRIPTION
**Pull request by Jackyuanx**
## Overview: What does this pull request change?
This closes [#3098](https://github.com/ManimCommunity/manim/issues/3098) by referencing the moving_camera_scene module documentation, and adding an additional example provided by MrDiver

## Motivation and Explanation: Why and how do your changes improve the library?
This closes [#3098](https://github.com/ManimCommunity/manim/issues/3098)

## Links to added or changed documentation pages

This affects [MovingCameraScene](https://docs.manim.community/en/stable/reference/manim.scene.moving_camera_scene.MovingCameraScene.html#manim.scene.moving_camera_scene.MovingCameraScene) and [moving_camera_scene](https://docs.manim.community/en/stable/reference/manim.scene.moving_camera_scene.html)

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/scene/moving_camera_scene.py [hunk 1]
```diff
@@ -83,8 +102,10 @@ class MovingCameraScene(Scene):
     This is a Scene, with special configurations and properties that
     make it suitable for cases where the camera must be moved around.
 
+    Note: Examples are included in the moving_camera_scene module
+    documentation, see below in the 'see also' section.
     .. SEEALSO::
-
+        :mod:`.moving_camera_scene`
```
Comment (JasonGrace2282) [inline manim/scene/moving_camera_scene.py:108]: This renders incorrectly, we might need an extra line before and after the `.. SEEALSO::`

**Review by JasonGrace2282 — CHANGES_REQUESTED** [review]
Overall looks good to me, one minor comment, and then I believe it's good to go.
Thanks for helping make Manim better :sparkles:

File: manim/scene/moving_camera_scene.py (hunk 1 above)
Comment (Jackyuanx) [reply manim/scene/moving_camera_scene.py]: Thanks, I have added the change

Comment (TahitiX) [reply manim/scene/moving_camera_scene.py]: Thank you! We have added extra line before and after `.. SEEALSO::`. 
Is it okay now?

**Comment by TahitiX** [comment]
Thank you! We have added extra line before and after .. SEEALSO::.
Is it okay now?

**Review by JasonGrace2282 — APPROVED** [review]
Thanks, it looks good to me! Thanks for adding this example.

**Pull request merged**